www/caddy/src/opnsense/mvc/app/models/Pischem/Caddy/FieldTypes/RealInterfaceField.php

```diff
@@ -0,0 +1,134 @@
+<?php
```

Comment:
I'm not sure why this is needed and it's also fragile as interface assignments change. The template can translate the interface to the device ("real interrface") name.

Reply:
Like this:
https://github.com/opnsense/plugins/blob/master/net/frr/src/opnsense/service/templates/OPNsense/Quagga/bgpd.conf#L2
https://github.com/opnsense/plugins/blob/master/net/frr/src/opnsense/service/templates/OPNsense/Quagga/bgpd.conf#L79

Just grep for physical_interface in plugins code, there are plenty of examples.

Reply:
Solved by changing the Caddyfile template for physical_interface usage.